task/kubeconfig-creator/0.1/kubeconfig-creator.yaml

```diff
@@ -61,7 +61,7 @@ spec:
     - name: output
   steps:
     - name: write
-      image: gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/kubeconfigwriter:v0.16.2
+      image: gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/kubeconfigwriter:v0.20.1
```

Comment:
can we tag the image with digest as well?

Reply:
done